tutorials/03_thermal_camera.md

```diff
@@ -94,7 +95,12 @@ This is currently unused by Ignition Gazebo.
 Now that we have set up our thermal camera, we'll need to assign temperatures to models in the environment.
 If a model doesn't have a temperature associated with it, then the thermal camera cannot detect it.
```

Comment:
new behavior in Dome:

```
If a model doesn't have a temperature associated with it, then it will be assigned the ambient temperature. There will some variation in their temperature based on color.
```

Reply:
https://github.com/ignitionrobotics/ign-sensors/pull/79/commits/fadf90b08ccb974c40e841bda92a5601ce2666b5